code_to_gast/routers/js_router.py

```diff
@@ -56,6 +58,9 @@ def node_to_gast(node):
     # Conditionals
     elif node.type == "IfStatement":
         return js_conditional.if_statement_to_gast(node)
+    # Loops
+    elif node.type == "WhileStatement":
```

Comment:
It would be nice to use enums here similar to the py router. Helps avoid typo bugs. For example, I noticed WhileStatement statement here has a capital W while js_loop.py uses lowercase w.

Reply:
Yeah- I think the reason we did it that way in py_router is just because of the way the python ast library works. We would prob have to create a custom Enum which I've never done before but definitely could try to figure it out. Probably would be another PR though

Reply:
Added to backlog!